spec/services/episode_services/clean_index_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe EpisodeServices::CleanIndex do
+  let(:service) { EpisodeServices::CleanIndex.new.call }
+  let!(:episodes) do
+  [
+    Fabricate(:episode, indexed_at: 1.minute.ago),
+    Fabricate(:episode, indexed_at: 1.day.ago)
+  ]
+  end
+
+  it "runs indexer for all outdated the episodes" do
+    service
```

Comment:
I would change this to `service.call`, leaving the `service` let statement as an object instance of the `CleanIndex` service.

Inside the blocks is nice to explicitly have verbs representing the intention of the test. A nice way to think of that is "Given, When, Then".

e.g.:

**Given** "the service instance & and episodes" -> implicit, not a huge issue since it is shared and readable
**When** "I *call* the service" -> explicit with "service.call"
**Then** "I *expect* the indexes to be empty" ...

If you like to take a further read on the subject:
https://martinfowler.com/bliki/GivenWhenThen.html

Reply:
This "Given, When, Then" way of thinking really helps!

Reply:
https://martinfowler.com/bliki/GivenWhenThen.html